Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -405,19 +405,21 @@ def incident_add_comment_command(client, args):
 
 def get_entity_by_id_command(client, args):
     entity_id = args.get('entity_id')
-    url_suffix = f'entities/{entity_id}'
-
-    result = client.http_request('GET', url_suffix, is_get_entity_cmd=True)
+    expansion_id = "98b974fd-cc64-48b8-9bd0-3a209f5b944b"
+    url_suffix = f'entities/{entity_id}/expand'
+    data = {
+        "expansionId": expansion_id
+    }
 
-    flattened_result = flatten_entity_attributes(result)
+    result = client.http_request('POST', url_suffix, is_get_entity_cmd=True, data=data)
```

Comment:
Shouldn't you combine the new field with the previous values? 
Are you currently just throwing the full object into the context?

Reply:
Moreover, why didn't we modify the yml as well?

Reply:
We do not have events in the system to check what is coming back.
That's why I'm not entirely sure of my answers.
I think the current implementation also has the information that was in the previous implementation and more information.

Reply:
Why did you resolve this? @ChanochShayner 
@dantavori Would love your advice here

Reply:
@ChanochShayner this might help:
* current command's outputs - https://github.com/Azure/azure-rest-api-specs/blob/master/specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/examples/entities/GetSecurityAlertEntityById.json
* `/expand` outputs - https://github.com/Azure/azure-rest-api-specs/blob/master/specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/examples/entities/expand/PostExpandEntity.json

note that the `/expand` example it's an *account* entity (and not a security alert), but if the structure is the same for every entity type then it looks like the two responses above have the same structure (id, name, type, kind and properties) and what differentiates them is the "properties" value (I guess each one has different attributes). So maybe the right thing to do is to combine the properties from the two responses.

However, this is just an advice and the best thing to do is to try the endpoint with data. Did you manage to get something with @Arsenikr ?